python/ray/_private/utils.py

```diff
@@ -350,18 +365,63 @@ def set_cuda_visible_devices(gpu_ids):
     Args:
         gpu_ids (List[str]): List of strings representing GPU IDs.
     """
-
     if os.environ.get(ray_constants.NOSET_CUDA_VISIBLE_DEVICES_ENV_VAR):
         return
-
     global last_set_gpu_ids
     if last_set_gpu_ids == gpu_ids:
         return  # optimization: already set
-
-    os.environ["CUDA_VISIBLE_DEVICES"] = ",".join([str(i) for i in gpu_ids])
+    set_visible_ids(gpu_ids, "CUDA_VISIBLE_DEVICES")
     last_set_gpu_ids = gpu_ids
 
 
+def set_aws_neuron_core_visible_ids(neuron_core_ids):
+    """Set the NEURON_RT_VISIBLE_CORES environment variable.
+
+    Args:
+        neuron_core_ids (List[str]): List of strings representing core IDs.
+    """
+    if os.environ.get(ray_constants.NOSET_AWS_NEURON_VISIBLE_CORES_ENV_VAR):
```

Comment:
We should allow neuron_core_ids == None case? (and it should be none when there's no neuron core in a node)?